src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -62,6 +65,7 @@ export const PublicRegistrationContent = ({ registration }: PublicRegistrationCo
       <Helmet>
         <title>{mainTitle}</title>
       </Helmet>
+      <DeletedPublicationScreenReaderInformation registration={registration} />
```

Comment:
bra du tenker på skjermlesere her! (jeg hadde neppe kommet på å tenkt på det i dette tilfellet 🙃)

Reply:
Det var en stygg banner midt i publikasjonen, veldig kjipt å få opplest halve greia, og så komme til "ååå forresten denne greia er sletta forøvrig".